requirements/base.txt

```diff
@@ -313,7 +313,7 @@ werkzeug==2.3.2
     #   flask
     #   flask-jwt-extended
     #   flask-login
-wrapt==1.15.0
+wrapt==1.12.0
```

Comment:
Conflicting dependency, I didn't find wrapt used as a dependency anywhere so I'm not sure if the versioning matters?

Reply:
You are not supposed to manually change the txt files. The dependencies in base.txt are controlled via setup.py.  You can see the instructions on how to regenerate the txt files at CONTRIBUTING.md.

In a nutshell:
```
$ python3 -m venv venv
$ source venv/bin/activate
$ python3 -m pip install -r requirements/integration.txt
$ pip-compile-multi --no-upgrade
```